caravel/views.py

```diff
@@ -1793,6 +1793,12 @@ def dashboard(self, dashboard_id):
         def dashboard(**kwargs):  # noqa
             pass
         dashboard(dashboard_id=dash.id)
+        if request.args.get("standalone") == "true":
+            return self.render_template(
+                "caravel/dashboard_standalone.html",
```

Comment:
I saw that we have a separate standalone template for explore view: https://github.com/ascott/caravel/blob/f837733d858643152304469c899c096e31d2bf24/caravel/views.py#L1194

Apart from the header, I also took out the button groups, fav star to limit interactions on dashboard standalone page. We could have if statements in both basic.html and dashboard.html for rendering specific parts based on standalone argument, but I was thinking that having a separate template would be cleaner?